test/integration/manifests/datapath/linux-deployment.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
```

Comment:
can we not reuse test\integration\manifests\goldpinger\deployment.yaml ?

Reply:
The original goldpinger deployment yaml does not have:
```
              - name: HOSTS_TO_RESOLVE
                value: "1.1.1.1 8.8.8.8 www.bing.com"
```
That we need to use to test host resolution